nautobot_ssot/jobs/base.py

```diff
@@ -47,17 +48,56 @@ class DataSyncBaseJob(BaseJob):
 
     dry_run = BooleanVar()
 
+    def load_source_adapter(self):
+        """TODO:."""
+        pass
+
+    def load_target_adapter(self):
```

Comment:
We should establish a consistent expectation here - should these methods take the source and target adapter as inputs? Or are they expected to construct the source/target adapter as part of the method? Or is the expectation that we'll already have `self.source_adapter` and `self.target_adapter` as instance properties? We should pick one and establish it as the standard.

Reply:
My preference would be to have `self.source_adapter` and `self.target_adapter` as instance properties. I think it would make it easier to provide default method if we can access the adapters from the instance directly